lib/node_modules/@stdlib/stats/base/dvarmpn/benchmark/c/benchmark.length.c

```diff
@@ -120,6 +121,40 @@ static double benchmark( int iterations, int len ) {
 	return elapsed;
 }
 
+/**
+* Runs a benchmark.
+*
+* @param iterations   number of iterations
+* @param len          array length
+* @return             elapsed time in seconds
+*/
+static double benchmark2( int iterations, int len ) {
+	double elapsed;
+	double x[ len ];
+	float v;
+	double t;
+	int i;
+
+	for ( i = 0; i < len; i++ ) {
+		x[ i ] = i;
+	}
```

Comment:
```suggestion
	for ( i = 0; i < len; i++ ) {
		x[ i ] = ( rand_double() * 20000.0 ) - 10000.0;
	}
```
we don't need a sorted array here, so better to follow the same convention as the non-offset interface

Reply:
Resolved